docs/ref/class-based-views/generic-date-based.txt

```diff
@@ -591,11 +591,11 @@ views for displaying drilldown pages for date-based data.
 
     **Example myapp/urls.py**::
 
-        from django.conf.urls import url
+        from django.urls import path
         from django.views.generic.dates import DateDetailView
 
         urlpatterns = [
-            url(r'^(?P<year>[0-9]{4})/(?P<month>[-\w]+)/(?P<day>[0-9]+)/(?P<pk>[0-9]+)/$',
+            path('<int:year>/<int:month>/<int:day>/<int:pk>/',
```

Comment:
original matched "nov" as well, <int:> does not.

Also happy to see this validates the "day or pk" discussion we had earlier.